net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -334,9 +369,26 @@ if /bin/pgrep -q ospfd; then
 	sleep 1
 	/usr/bin/killall -9 ospfd 2>/dev/null
 fi
+if /bin/pgrep -q ospf6d; then
+        /usr/bin/killall ospf6d
+        sleep 1
+        /usr/bin/killall -9 ospf6d 2>/dev/null
+fi
+if /bin/pgrep -q bgpd; then
+	/usr/bin/killall bgpd
+	sleep 1
+	/usr/bin/killall -9 bgpd 2>/dev/null
+fi
 {$carp_ip_status_check}
+/usr/bin/touch {$quagga_config_base}/zebra.conf
```

Comment:
What is the purpose of all these touch commands? If the pkg code writes out a valid configuration, it will be present and the touch will not help anything. If the file is not there, the daemon shouldn't try to start. (Same for this and the three lines after)